### DISCUSSION TITLE
practicas mision front-end js

### DISCUSSION THREAD

**Discussion by LuisAntonioLinaresCarbajal** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#11] [Q&A] [answered]
¿Alguien sabe donde estarán disponibles las practicas de frontend ?

**Comment by doguedogue** [comment]
Las van a subir en el teams
Equipos --> Front End JS --> semana 1

**Reply by Ingrzs** [reply]
Y creo todavía no han subido nada, es cuestión de estar checando 😅

**Reply by RodolfoMorquecho** [reply]
disculpa, ayer comento el MC que subiría un video para realizar ejercicios pero no lo encuentro, sabes donde lo puedo encontrar o tal vez entendí mal.

**Reply by GloriaCoralCerecedo** [reply]
Hola, si mencionó sobre el video y dijo que lo iba a subir hoy

**Reply by RodolfoMorquecho** [reply]
ok mas tarde reviso, gracias Gloria.

**Reply by GloriaCoralCerecedo** [reply]
Denada 😁

**Reply by IsaiCeSa** [reply]
Hasta ahora no hay ejercicios para esta primera semana, ¿pueden confirmar?

**Reply by xhanzary9** [reply]
Confirmo :/

**Reply by LuisAntonioLinaresCarbajal** [reply]
F

**Reply by JoseAngelSil** [reply]
una duda existencial. hoy no hay sesion con el MC?

**Reply by EdgarValenciaR** [reply]
Hola @JoseAngelSil no amigo, las sesiones solo son lunes, miercoles y jueves. Saludos!!

**Comment by romarpla** [comment, marked as answer]
Los ejercicios de la primer semana se subirán hoy Viernes 18/02/22

**Reply by humpeguz** [reply]
y como se hacen :( porque no estoy entendiendo nada
